src/main/java/com/autosale/shop/service/impl/SecurityServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.autosale.shop.service.impl;
+
+import com.autosale.shop.model.JwtTokensDAO;
+import com.autosale.shop.model.User;
+import com.autosale.shop.service.JwtTokenService;
+import com.autosale.shop.service.SecurityService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class SecurityServiceImpl implements SecurityService {
```

Comment:
What if the sense of this service? Why didn't you add some additional logic here?  Probably it worth combining UserService and JwtTokenService here? And split responsibilities

Reply:
I can't imagine how to do it, what exactly and where should be separated, maybe this service is unnecessary for me and it can be deleted if it is not burdened with any logic?

Reply:
if I split UserService and JwtTokenService and combine them here then this will break the responsibility zone and I will have to pass the service as a parameter to the method which I don't think is the best idea.